packages/server/api/controllers/orders.controller.js

```diff
@@ -0,0 +1,38 @@
+const knex = require('../../config/db');
+const HttpError = require('../lib/utils/http-error');
+
+const getOrderItemsByUserId = async (token) => {
+  try {
+    if (!token) {
+      throw new HttpError('You are not authenticated', 401);
+    }
+
+    const userUid = token.split(' ')[1];
+
+    if (!userUid) {
+      throw new HttpError('Token is wrong', 400);
+    }
+
+    const user = (await knex('users').where({ uid: userUid }))[0];
+
+    if (!user) {
+      throw new HttpError('User was not found', 404);
+    }
+
+    const orderItems = await knex('orders')
+      .where({
+        user_id: user.id,
```

Comment:
Why do we need to first resolve the user to get their ID? Doesn't the endpoint accept the `userId` already? Why would it change in between?